lib/tzdata/data_loader.ex

```diff
@@ -4,8 +4,15 @@ defmodule Tzdata.DataLoader do
   require Logger
   # Can poll for newest version of tz data and can download
   # and extract it.
-  @download_url "https://data.iana.org/time-zones/tzdata-latest.tar.gz"
-  def download_new(url \\ @download_url) do
+  def download_url do
+    Application.get_env(:tzdata, :download_url, @download_url)
+  end
```

Comment:
Now the `@download_url` is missing @rai200890  :sweat_smile: 
```suggestion
  @download_url "https://data.iana.org/time-zones/tzdata-latest.tar.gz"
  def download_url do
    Application.get_env(:tzdata, :download_url, @download_url)
  end
```